libs/design/src/scss/typography/_mixins.scss

```diff
@@ -1,11 +1,33 @@
+/**
+* @docs
+* 
+* Changes the weight (boldness) of text to 700
+*
+* @usage @include embolden(500);
+*/
 @mixin embolden($weight: 700) {
     font-weight: $weight;
 }
 
+/**
+* @docs
+* 
+* Changes the casing of a piece of text to uppercase
+*
+* @usage @include uppercase();
+*/
 @mixin uppercase() {
     text-transform: uppercase;
 }
 
+/**
+* @docs
+* 
+* Forces a line of test (with a fixed with) to ellipsis 
```

Comment:
Text  not test

Reply:
Resolved in 6e0d8e5f09e11ac7c0fd9050ac33eb6a213be97d